### ISSUE TITLE
Ensure Search Terminology is Consistent for Cljdoc Users

### ISSUE THREAD

**Issue by lread** [cljdoc/cljdoc#571]
## Currently
Cljdoc only offers the ability to search for libraries.

## But...
@corasaurus-hex will be adding the ability to search within a single library's documentation.

When chatting, Cora and I had been describing these by the details of their implementations:
- server-side search for libraries
- client-side search for single library search

But this doesn't help the cljdoc user at all.

## So...
We had a chat on Slack and came up with the following:

- Cljdoc already uses the term **docset**. We are not sure where this term comes from, but see it used in [dash](https://kapeli.com/dash) and [doxygen](https://www.doxygen.nl). As long as we make clear what a docset is to our users, we think it is just fine.
- We'll call our current search feature **libraries search**
- We'll call our new search feature **single docset search**. Once a user understands what a docset is, this describes the feature clearly and opens us up for some possible future "multi docset search" feature.

## Actions
- Review user-facing docs and UI to ensure we define and use **docset** consistently.
- Before (or soon after) the single docset search feature is released, ensure that docs and UI use/convey **single docset search** and **libraries search** consistently.

**Comment by corasaurus-hex** [comment]
Some definitions:

A **docset** is the generated documentation for a single release of a library. A **docset** is located and identified by its **coordinate**. The **coordinate** of a **docset** is formatted as `group-id/artifact-id/version`. For example, the **coordinate** for the latest version (at time of writing) of `clj-common`'s `fs` library is `clj-commons/fs/1.6.310`. You can use this **coordinate** in the URL to load the **docset** [https://cljdoc.org/d/clj-commons/fs/1.6.310](https://cljdoc.org/d/clj-commons/fs/1.6.310).

**Comment by lread** [comment]
Thanks for adding definitions!

Is a dash-flavored docset different than our [current downloadable docset](https://github.com/cljdoc/cljdoc/blob/master/doc/userguide/for-users.md#offline-docs)?

**Comment by corasaurus-hex** [comment]
Oh cool, I wasn't aware this already worked!

**Comment by lread** [comment]
Yeah, I've been dabbling in cljdoc for a while now and still often find it-does-that?!s.
